app/lib/activitypub/activity/like.rb

```diff
@@ -3,12 +3,39 @@
 class ActivityPub::Activity::Like < ActivityPub::Activity
   def perform
     original_status = status_from_uri(object_uri)
+    return if original_status.nil? || !original_status.account.local? || delete_arrived_first?(@json['id'])
 
-    return if original_status.nil? || !original_status.account.local? || delete_arrived_first?(@json['id']) || @account.favourited?(original_status)
+    return if maybe_process_misskey_reaction
```

Comment:
nit: with the addition of non-misskey react content (generalised in spec), maybe this one could be `maybe_process_embedded_reaction`?

Reply:
4f15150fa1c71e0db9329c53ed069938852317aa does this.